backend/src/User.ts

```diff
@@ -0,0 +1,66 @@
+import { UserDBEntity } from './entities/UserDBEntity';
+import * as bcrypt from 'bcrypt';
+
+export enum AUTH_RESULT_CODE {
```

Comment:
all capslocks are reserved for constants. This isnt a constant, it is a enum (which is more like a class/type). Use camel case instead.